azure-mgmt-appservice/src/main/java/com/azure/management/appservice/ManagedServiceIdentity.java

```diff
@@ -14,7 +14,7 @@
  * The ManagedServiceIdentity model.
  */
 @Fluent
-public final class ManagedServiceIdentity {
+public class ManagedServiceIdentity {
```

Comment:
Do the tests pass when JsonInclude is not added for the map? At Compute side, I have to add that.

Reply:
Yes it passes.

I think for JsonInclude it more likely for a bug on service. Service must have this field even when it is nothing there. So JsonInclude there to ensure that. Unless the field is marked as `required`, I see no reason for service to do this.